app/controllers/guests_controller.rb

```diff
@@ -64,16 +67,45 @@ def set_guest
 
   def notify_host
     host = @guest.event.host
-    notification = GuestNotification.with(type: "join", host: host,
-                                          recipient: @guest.guest, status: @guest.status, event: @guest.event)
-    pp notification
+    notification = GuestNotification.with(
+      type: "event",
+      message: "#{@guest.guest.first_name} would like to join your event.",
+      recipient: @guest.guest,
+      link_to: @guest.event
+    )
     notification.deliver(host)
   end
 
   def notify_recipient
     guest = @guest.guest
-    notification = GuestNotification.with(type: "invite", host: @guest.event.host,
-                                          recipient: guest, status: @guest.status, event: @guest.event)
+    notification = GuestNotification.with(
+      type: "event",
+      message: "#{@guest.event.host.first_name} invited you to their event.",
+      recipient: @guest.guest,
+      link_to: @guest.event
+    )
     notification.deliver(guest)
   end
+
+  def notify_guest_accept
+    guest = @guest.guest
+    notification = GuestNotification.with(
+      type: "event",
+      message: "#{@guest.event.host.first_name} accepted your request to join this event.",
+      recipient: @guest.guest,
```

Comment:
Can they be replaced by `recipient: guest` when it's the same as `@guest.guest`?